lib/jekyll/commands/new_theme.rb

```diff
@@ -22,14 +22,20 @@ def process(args, opts)
       end
 
       new_theme_name = args.join("_")
-      theme = Jekyll::ThemeBuilder.new(new_theme_name, opts)
+      @theme = Jekyll::ThemeBuilder.new(new_theme_name, opts)
+
+      update_logger
+    end
+
+    def update_logger
```

Comment:
I didn't notice this before, but the addition of the `update_logger` method complicates this command quite a bit. I think we're better off without the extra method. Please make that change, and I'll reapprove.